src/components/extensive/Modal/ModalAdd.tsx

```diff
@@ -62,14 +63,26 @@ const ModalAdd: FC<ModalAddProps> = ({
         <Icon name={IconNames.WRI_LOGO} width={108} height={30} className="min-w-[108px]" />
         <div className="flex items-center">
           <When condition={status}>
-            <Status status={status ? status : "draft"} className="rounded px-2 py-[2px]" textVariant="text-14-bold" />
+            <Status
+              status={(status ? status : "draft") as StatusEnum}
+              className="rounded px-2 py-[2px]"
+              textVariant="text-14-bold"
+            />
           </When>
           <button onClick={onClose} className="ml-2 rounded p-1 hover:bg-grey-800">
             <Icon name={IconNames.CLEAR} width={16} height={16} className="text-darkCustom-100" />
           </button>
         </div>
       </header>
       <div className="max-h-[100%] w-full overflow-auto px-8 py-8">
+        <When condition={!!iconProps}>
+          <Icon
+            {...iconProps!}
+            width={iconProps?.width || 40}
+            className={twMerge("mb-8", iconProps?.className)}
+            style={{ minHeight: iconProps?.height || iconProps?.width || 40 }}
```

Comment:
need the `??` treatment here and in `ModalApprove` as well.

Reply:
done

Reply:
`ModalApprove` still appears to need an update.